README.md

```diff
@@ -2,6 +2,15 @@
 
 ![Build](https://github.com/NHSDigital/eps-prescription-status-update-api/workflows/release/badge.svg?branch=main)
```

Comment:
This looks like a duplicate of the release badge on line 6 and links to a 404. Should we remove it?

Reply:
have removed the duplicate. It points to a 404 currently as the workflow has not run yet and will not until this branch is merged